pyUltroid/__main__.py

```
@@ -18,17 +18,16 @@ def main():
         WasItRestart,
         autopilot,
         customize,
-        fetch_ann,
         plug,
         ready,
         startup_stuff,
     )
     from .startup.loader import load_other_plugins
 
     try:
-        from apscheduler.schedulers.asyncio import AsyncIOScheduler
+        pass
```

why try..except pass ?

---

I can re enable this, but I turned the schedulers off to reduce latency and when and if Ultroids fetch ann is still down .

---

and pylint kept doing it disabled now

---

uhm, resolve it

---

wth i did.... :S